tools/src/main/java/org/apache/kafka/trogdor/workload/ConnectionStressWorker.java

```diff
@@ -112,6 +112,18 @@ public void start(Platform platform, WorkerStatusTracker status,
         }
     }
 
+    /**
+     * Update the worker's status.
+     * This method should be called inside a lock on the ConnectionStressWorker object
+     */
+    private void updateStatus(long lastTimeMs) {
```

Comment:
This function should be synchronized since it accesses totalConnections, totalFailedConnections, etc. which are only ever accessed under the lock.